barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/goblin_verifier.test.cpp

```diff
@@ -16,35 +16,27 @@ namespace bb::stdlib::recursion::honk {
  *
  * @tparam Builder Circuit builder for the recursive verifier circuit
  */
-template <typename BuilderType> class GoblinRecursiveVerifierTest : public testing::Test {
-
-    using UltraComposer = UltraComposer_<UltraFlavor>;
-    using GoblinUltraComposer = UltraComposer_<GoblinUltraFlavor>;
+template <typename OuterFlavor> class GoblinRecursiveVerifierTest : public testing::Test {
```

Comment:
It's tempting to template by `OuterVerifier` and then extract the other types via the chain `Verifier -> Prover -> Flavor -> Builder` but it didn't seem necessary. The cost is that this code isn't generic in the flavor since we have hard-coded `UltraFoo_`s, but it doesn't need to be more generic.